ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -353,6 +356,25 @@ S3Open(WT_FILE_SYSTEM *fileSystem, WT_SESSION *session, const char *name,
     return (0);
 }
 
+/*
+ * S3Size --
+ *     Get the size of a file in bytes, by file name.
+ */
+static int
+S3Size(WT_FILE_SYSTEM *fileSystem, WT_SESSION *session, const char *name, wt_off_t *sizep)
+{
+    S3_STORAGE *s3;
+    s3 = FS2S3(fileSystem);
```

Comment:
Can we combine this assignment and declaration on one line instead?

Reply:
Yes we can!